Sam 259 requests without via

nunomsfranco opened the pull request:
- Added more REST methods for requests without VIA connection (get-type and convert). After conversion the new converted file is sent to VIA before the task its given as finished.
- Added more flags to the ghostscript to increase performance and added a pdfa_def.ps file to grantee that  converted file is pdf/a.
- Other minor fixes.

nunomsfranco committed:
Added more URI for requests without VIA

nunomsfranco committed:
converted file is sent to VIA before its given as finished
- added more flags to the ghostscript to increase performance
- added pdfa_def.ps file to garantee that  converted file is pdf/a
- minor fixes

numsu commented on docsbox/docs/views.py:
```diff
@@ -31,9 +29,22 @@ def get(self, task_id):
+        else:
+            return abort(400, message="file field is required")
+
+class DocumentTypeFromIdView(Resource):        
```
Do these need to be separate functions?

Take in the file and the id, if there is an id, get it from VIA, if not, use the file - if neither -> return 400.

Same goes for the other functions.

nunomsfranco replied:
Fixed - I've joined the functions

numsu commented on nginx.conf:
```diff
@@ -27,6 +27,11 @@ http {
             proxy_set_header X-Real-IP $remote_addr;
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            
+            proxy_connect_timeout 120;
```
Also add the other changes we've made to the nginx.conf on the server

nunomsfranco committed:
fixes
- joined similar requests may recebve file_id and/or file
- added the option for receving the via allowed users through headers-
- changed nginx.conf to be in par with the test env

nunomsfranco committed:
- if via-allowed-users header isnt given use default

nunomsfranco committed:
small fix
